code/modules/mapping/README.txt

```diff
@@ -1,5 +1,5 @@
 The code in this module originally evolved from dmm_suite and has since been
-specialized for SS13 and otherwise tweaked to fit /tg/station's needs.
```

Comment:
What I said for bug_report.md's 9th, feedback_conversion_2017-11-12.py's 493rd, wrestling.dm's 3rd, /lua/README.md's 93rd and 124th lines applies for this line.